### DISCUSSION TITLE
how to install indy10 on the latest version of Delphi?

### DISCUSSION THREAD

**Discussion by starwindz** [IndySockets/Indy#610] [Q&A] [answered]
How to install indy10 on the latest version of Delphi after uninstalling built-in Indy? Which (package) files need to be used specifically?

**Comment by rlebeau** [comment, marked as answer]
Have you read the documentation in this very same repo yet?
https://github.com/IndySockets/Indy/wiki/Updating-Indy

**Comment by starwindz** [comment]
sorry for my mistake. following wiki has all info about uninstaling built-in version of Inday and how to install the latest Indy. somewhat confused with Delphi version number but the wiki has exact info
https://github.com/IndySockets/Indy/wiki/Updating-Indy
